docs/data/charts/legend/legend.md

```diff
@@ -12,28 +12,35 @@ In chart components, the legend links series with `label` properties and their c
 
 {{"demo": "BasicLegend.js"}}
 
-## Placement
+## Customization
+
+### Position
 
 The legend can either be displayed in a `'column'` or `'row'` layout controlled with the `direction` property.
 
-It can also be moved by a combination of `position: { vertical, horizontal }` properties and the legend offset values.
-The `position` places the legend just next to the drawing area, and offset values move it relative to this base position.
+It can also be moved with the `position: { vertical, horizontal }` property which defines how the legend aligns within the SVG.
 
 - `vertical` can be `'top'`, `'middle'`, or `'bottom'`.
 - `horizontal` can be `'left'`, `'middle'`, or `'right'`.
-- offsets are set with CSS variables `--ChartsLegend-rootOffsetX` and `--ChartsLegend-rootOffsetY`.
+
+You can add some space to a given `position` with a `padding` property which can be either a number or an object `{ top, bottom, left, right }`.
+This `padding` will add space between the SVG borders and the legend.
```

Comment:
I am not entirely sure about the phrasing of 'a given position'. It makes sense to me, but I'm not sure it is intelligible at a first glance. I don't have a better suggestion for this though 🙈 🤔 
```suggestion
You can add spacing to a given `position` with the `padding` property, which can be either a number or an object `{ top, bottom, left, right }`.
This `padding` will add space between the SVG borders and the legend.
```